scaffolds/gatsby/scaffold.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Template, Zombi } from 'compiled/zombi';
+import { createScaffold } from 'cli/utils/scaffold-helpers';
+import { mergePrompts } from 'cli/utils/merge-prompts';
+import { NpmClientPrompt, PublishableApiKeyPrompt } from 'scaffolds/prompts';
+
+type GatsbyData = NpmClientPrompt.Data & PublishableApiKeyPrompt.Data;
+
+export default createScaffold<GatsbyData>(
+  (props) => (
+    <Zombi {...props} prompts={mergePrompts(PublishableApiKeyPrompt.questions, NpmClientPrompt.questions)}>
+      <Template source="./src/components" />
+      <Template source="./src/images" data={false} />
+      <Template source="./src/lib" />
+      <Template source="./src/pages" />
+      <Template source="./src/util" />
+      <Template source="./.env.development" />
+      <Template source="./.gitignore" />
+      <Template source="./.prettierrc" />
+      <Template source="./.prettierignore" />
+      <Template source="./gatsby-browser.js" />
+      <Template source="./gatsby-config.js" />
+      <Template source="./gatsby-node.js" />
+      <Template source="./gatsby-ssr.js" />
+      <Template source="./LICENSE" />
+      <Template source="./README.md" />
+      <Template source="./package-lock.json" />
```

Comment:
Let's please remove `package-lock.json`. For JS projects, we don't include the dependency lock file because the user may choose their NPM client.

Reply:
Sure. 

Make sense to not include the lock file.